snippets/sdn-deprecation-statement.adoc

```diff
@@ -0,0 +1,27 @@
+// Text snippet included in the following assemblies:
+//
+// * networking/about-networking.adoc
```

Comment:
I was unable to open the following assemblies: 
![Screenshot from 2024-04-05 16-15-56](https://github.com/openshift/openshift-docs/assets/57954076/b13b09b2-b92f-4149-b75a-774f8d943acf)
![Screenshot from 2024-04-05 16-17-23](https://github.com/openshift/openshift-docs/assets/57954076/e7399ffe-b5d8-4b7f-a004-4d3bb66c3a33)

Reply:
https://73316--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/about-networking seems to render fine on my end